scipy/io/matlab/mio.py

```diff
@@ -157,6 +157,52 @@ def loadmat(file_name, mdict=None, appendmat=True, **kwargs):
     >>> mat_contents['testdouble']
     array([[0.        , 0.78539816, 1.57079633, 2.35619449, 3.14159265,
             3.92699082, 4.71238898, 5.49778714, 6.28318531]])
+
+    By default SciPy reads MATLAB structs as structured NumPy arrays where the
+    dtypes are objects named after the MATLAB struct fields. This can be
```

Comment:
I'm really sorry, I see what you mean now, but I think this would be better phrased as "where the dtype field names correspond to the MATLAB struct field names, and each field is of `object` type".

Reply:
By the way, I think the reason for this is because there is no requirement for the field values to be of the same type across structs in a Matlab struct array:

```matlab
>> s = struct('foo', {'bar', 1});
>> s.foo
ans = bar
ans =  1
```

Reply:
Yes I agree, I think that this *is* the reason that the `dtype` of the MATLAB fields is `object`. Thanks so much for your review, and for letting me piggy back onto your PR. I hope that I've managed to add some value and not detract too much from your original intent.